$OpenBSD: patch-ext_aes_meson_build,v 1.1 2022/02/05 14:49:40 ajacoutot Exp $

XXX until our libssl.pc exposes a higher version
ext/aes/meson.build:9:0: ERROR: Invalid version of dependency,
need 'openssl' ['>= 1.1.0'] found '1.0.0'.

Index: ext/aes/meson.build
--- ext/aes/meson.build.orig
+++ ext/aes/meson.build
@@ -6,7 +6,7 @@ aes_sources = [
 ]
 
 aes_cargs = []
-aes_dep = dependency('openssl', version : '>= 1.1.0', required : get_option('aes'))
+aes_dep = dependency('openssl', version : '>= 1.0.0', required : get_option('aes'))
 if aes_dep.found()
   aes_cargs += ['-DHAVE_OPENSSL']
 else
